src/Razor/Razor.Language/test/Microsoft.AspNetCore.Razor.Language.Test.csproj

```diff
@@ -1,9 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.2;net46</TargetFrameworks>
```

Comment:
nit: maybe make this net461?

Reply:
Razor intentionally targets net46, something about Visual Studio requirements...